arangod/RestHandler/RestViewHandler.cpp

```diff
@@ -62,7 +65,7 @@ RestViewHandler::RestViewHandler(GeneralRequest* request,
     : RestVocbaseBaseHandler(request, response) {}
 
 void RestViewHandler::getView(std::string const& nameOrId, bool detailed) {
-  auto view = _vocbase.lookupView(nameOrId);
+  auto view = CollectionNameResolver(_vocbase).getView(nameOrId);
```

Comment:
just a suggestion: Would it be useful to add a static helper method somewhere to find views (i.e. like the lookup function in `arangod/VocBase/Methods/Collections.cpp`).
It seems a little wasetful to always have to instantiate a CollectionNameResolver for this

Reply:
```
> Would it be useful to add a static helper method somewhere to find views 
that's kind of what  CollectionNameResolver does
```

Reply:
well no it has a cached of resolved collections etc, it just does much more